app/delivery/send_to_providers.py

```diff
@@ -115,6 +115,7 @@ def send_email_to_provider(notification):
             # Check if a MLWR sid exists
             if (
                 current_app.config["MLWR_HOST"]
+                and "https" in str(current_app.config["MLWR_HOST"])
```

Comment:
We could do a real url validation here but I don't think we currently have a library / function for this. Seems overkill to do a new import or complicated regex just for this.